beacon-chain/forkchoice/doubly-linked-tree/forkchoice.go

```diff
@@ -21,6 +21,8 @@ import (
 	"go.opencensus.io/trace"
 )
 
+const slotsPerEpoch = 32
```

Comment:
Needs to use config params

Reply:
> Needs to use config params

Can't use it. It needs to be constant. We probably can move these config params to constants but in another PR